docs/source/_src_getting_started/submodules.rst

```diff
@@ -0,0 +1,43 @@
+==================
+Tudatpy Submodules
+==================
+
```

Comment:
Don't have much comments on the following, but I think that it is a very good idea to document this :)